pkg/base1/cockpit.js

```diff
@@ -3486,14 +3486,18 @@ function full_scope(cockpit, $, po) {
 
         self.move = function move(beg, end) {
             stop_walking();
+            /* Some code paths use now twice.
+             * They should use the same value.
+             */
+            var now = $.now();
```

Comment:
I would prefer if this was only called when needed. The grid code is in a rather "hot" path performance wise.